src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -718,7 +718,7 @@ public void EmitCFICodes(int offset)
                     Debug.Assert(false);
                 }
 
-                if (_targetPlatform.OperatingSystem == TargetOS.OSX)
+                if (_targetPlatform.IsOSXLike)
                 {
                     // Emit a symbol for beginning of the frame. This is workaround for ld64
```

Comment:
@filipnavara based on this change: https://github.com/dotnet/runtime/commit/c8f9f2949a243c411bd05278efab3f5a857cd0c1 can this be considered applicable for all OSX-like architectures?

Reply:
Yep. 👍

Reply:
Thank you for the confirmation.